symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/application/ApplicationManagementService.java

```diff
@@ -0,0 +1,159 @@
+package com.symphony.bdk.core.service.application;
+
+import com.symphony.bdk.core.auth.AuthSession;
+import com.symphony.bdk.core.retry.RetryWithRecovery;
+import com.symphony.bdk.core.retry.RetryWithRecoveryBuilder;
+import com.symphony.bdk.core.util.function.SupplierWithApiException;
+import com.symphony.bdk.gen.api.AppEntitlementApi;
+import com.symphony.bdk.gen.api.ApplicationApi;
+import com.symphony.bdk.gen.api.model.ApplicationDetail;
+import com.symphony.bdk.gen.api.model.PodAppEntitlement;
+import com.symphony.bdk.gen.api.model.UserAppEntitlement;
+import com.symphony.bdk.http.api.ApiException;
+
+import org.apiguardian.api.API;
+
+import java.util.List;
+
+import javax.annotation.Nonnull;
+
+/**
+ * Service class for managing the applications and the application entitlements.
+ * <p>
+ * This services used for retrieving information about a particular application or application entitlements,
+ * performing some actions related to the applications like:
+ * <p><ul>
+ * <li>Create an application</li>
+ * <li>Update an existing application</li>
+ * <li>Delete an existing application</li>
+ * <li>Get the information of an existing application</li>
+ * <li>Update application entitlements</li>
+ * <li>Update user applications</li>
+ * </ul></p>
+ * </p>
+ */
+@API(status = API.Status.STABLE)
+public class ApplicationManagementService {
+
+  private final ApplicationApi applicationApi;
+  private final AppEntitlementApi appEntitlementApi;
+  private final AuthSession authSession;
+  private final RetryWithRecoveryBuilder<?> retryBuilder;
+
+  public ApplicationManagementService(ApplicationApi applicationApi, AppEntitlementApi appEntitlementApi,
+      AuthSession authSession, RetryWithRecoveryBuilder<?> retryBuilder) {
+    this.applicationApi = applicationApi;
+    this.appEntitlementApi = appEntitlementApi;
+    this.authSession = authSession;
+    this.retryBuilder = retryBuilder;
+  }
+
+  /**
+   * Create a new application.
+   *
+   * @param applicationDetail Contains the following fields for creating an application: appId, name, appUrl, domain, and publisher.
+   *                          Note that appUrl is not required.
+   * @return The created application.
+   * @see <a href="https://developers.symphony.com/restapi/v20.9/reference#create-app">Create Application</a>
```

Comment:
Here you don't need to target `v20.9` in particular, this endpoint is already available in current stable version : https://developers.symphony.com/restapi/reference#create-app